kubernetes/gaffer-jhub/docs/kind-deployment.md

```diff
@@ -0,0 +1,49 @@
+# Deploying JupyterHub for Gaffer using kind
```

Comment:
Would be nice if it explained how to view the example notebooks so they could act as a kind of tutorial / doc?

Reply:
👍 done